src/main/scala/viper/gobra/ast/frontend/AstPattern.scala

```diff
@@ -48,5 +48,5 @@ object AstPattern {
   case class Predicate(id: PIdnUse, symb: st.FPredicate) extends PredicateKind with Symbolic
   case class ReceivedPredicate(recv: PExpression, id: PIdnUse, path: Vector[MemberPath], symb: st.MPredicate) extends PredicateKind with Symbolic
   case class PredicateExpr(typ: PType, id: PIdnUse, path: Vector[MemberPath], symb: st.MPredicate) extends PredicateKind with Symbolic
-
+  case class PredExprInstance() extends PredicateKind
```

Comment:
I think that this should take the base expression of the predicate instance (predicate call) as a parameter.

Reply:
and the arguments of the instance are also missing.

Reply:
e.g. `case class PredExprInstance(base: PExpression, args: Vector[PExpression]) extends PredicateKind`

Reply:
I was a bit hesitant to add these two fields because I was always able to retrieve them in other ways but I agree that it is useful to have this information here